rstudio/rstudio
# Locale Cookie Handled by Electron

### Intent

Detect language changes from GWT and update Electron's texts based on it's set cookie.

### Approach

Fetching the cookies and setting them as a `Window LocalStorage Item` when a `BrowserWindow` is created or if the cookies are updated at any given point.

### Automated Tests

None

### QA Notes

Change the language by going to `Tools > Global Options > Advanced`. The screen should refresh. To test some other pages like Error, just give the `--session-exit` flag when running the app.

### Checklist
